inst/prelease.R

```diff
@@ -0,0 +1,115 @@
+# Open a pre-release issue on the dev_guide repo
+#
+# Uses the gh cli (https://cli.github.com), which must be installed and
+# authorised with 'gh auth'.
+#
+# The script calls the single function, `post_issue()`, defined at the end. All
+# functions defined prior to that are sub-functions used within that main
+# function.
+
+repo <- "https://github.com/ropensci/dev_guide"
+
+has_gh_cli <- function () {
+
+    ghs_path <- dirname (Sys.which ("gh"))
+    nzchar (ghs_path)
+}
+
+if (!has_gh_cli ()) {
+    stop ("The 'gh cli' must be installed; see https://cli.github.com")
```

Comment:
why not GitHub API?

Reply:
Because it would require likely hundreds more lines of code, plus the subjectively best thing is that that script is entirely base R :rocket:

Reply:
Oh, that's not quite true - it uses `jsonlite` just to save a few extra lines of code.

Reply:
:+1: